README.md

```diff
@@ -1,4 +1,4 @@
-# Percona Operator for PostgreSQL
+# Percona Operator for Ege
```

Comment:
You had a good mood :)

Reply:
one day I'll automate myself by writing a k8s operator :D